### ISSUE TITLE
Many Relu functions have been removed !!!????

### ISSUE THREAD

**Issue by ProWhalen** [CoinCheung/BiSeNet#6]
I have compared the model with origin author's. Many Relu functions have been removed !  Can anyone tell me why this is done ?

**Comment by jetxa** [comment]
Can you be more specific? Which module?

**Comment by ProWhalen** [comment]
you can compare the resnet18 with this one : https://github.com/pytorch/vision/blob/master/torchvision/models/resnet.py

**Comment by jetxa** [comment]
https://github.com/CoinCheung/BiSeNet/blob/master/resnet.py#L9
**BatchNorm2d** and **nn.BatchNorm2d** are different, BatchNorm2d contains the activation function

**Comment by CoinCheung** [comment]
@jetxa  Thanks for explaining, that is the reason why I removed many  Relu functions.

**Comment by CoinCheung** [comment]
@ProWhalen 
Hi, Just as what @jetxa explained, it is because I used the activated version of BatchNorm2d.
